pkg/cmd/projectconfig/update.go

```diff
@@ -23,7 +23,7 @@ var projectConfigUpdateCmd = &cobra.Command{
 	Args:  cobra.MaximumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		var projectConfig *apiclient.ProjectConfig
-		var projectConfigs []apiclient.CreateProjectConfigDTO
+		var projectDtos []apiclient.CreateProjectDTO
```

Comment:
Wasn't sure where to comment this so putting it here.

`CreateProjectDTO` used in all `projectConfig` files seems a bit wrong. Why don't we have `CreateProjectConfigDTO` anymore?